spec/duration.html

```diff
@@ -1207,17 +1211,16 @@ <h1>
         1. If _relativeTo_ is not present, set _relativeTo_ to *undefined*.
         1. If Type(_relativeTo_) is Object and _relativeTo_ has an [[InitializedTemporalZonedDateTime]] internal slot, then
           1. Let _endNs_ be ? AddZonedDateTime(_relativeTo_.[[Nanoseconds]], _relativeTo_.[[TimeZone]], _relativeTo_.[[Calendar]], 0, 0, 0, _days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_).
-          1. Set _nanoseconds_ to _endNs_ − _relativeTo_.[[Nanoseconds]].
+          1. Set _nanoseconds_ to ℝ(_endNs_ − _relativeTo_.[[Nanoseconds]]).
         1. Else,
-          1. Set _nanoseconds_ to ℤ(! TotalDurationNanoseconds(_days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_, 0)).
+          1. Set _nanoseconds_ to ! TotalDurationNanoseconds(_days_, _hours_, _minutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_, 0).
         1. If _largestUnit_ is one of *"year"*, *"month"*, *"week"*, or *"day"*, then
           1. Let _result_ be ? NanosecondsToDays(_nanoseconds_, _relativeTo_).
```

Comment:
```suggestion
          1. Let _result_ be ? NanosecondsToDays(ℤ(_nanoseconds_), _relativeTo_).
```
Alternatively - since NanosecondsToDays immediately converts to MVs, change it to expect one

Reply:
I made that change in #2102.